docs/userGuide/siteConfiguration.md

```diff
@@ -80,7 +80,7 @@ The prefix for all page titles. The separator <code>-</code> will be inserted by
 * **`src`**/**`glob`**: `src` can be used to specify a file e.g., `docs/index.md`.<br>
     Alternatively, `glob` can be used to define a file pattern in the [_glob syntax_](https://en.wikipedia.org/wiki/Glob_(programming)) e.g., `**/*.md`.
 * **`title`**: The page `<title>` for the generated web page. Titles specified here take priority over titles specified in the [front matter](addingPages.html#front-matter) of individual pages.
-* `layout`: The [layout]({{ baseUrl }}/userGuide/advancedTopics.html#page-layout) to be used by the page.
+* `layout`: The [layout]({{ baseUrl }}/userGuide/advancedTopics.html#page-layout) to be used by the page. Default: `default`.
 * `"searchable": "no"`: Specifies that the page(s) should be excluded from searching.

```

Comment:
`src` and `title` are bold. So `layout` and `searchable: "no"` should be as well since they belong to the same sub-section.

Small nit: only `"searchable"` has quotation marks while the rest does not.

Reply:
> Small nit: only `"searchable"` has quotation marks while the rest does not.

Updated that part. I was hesitant to say that the default for `searchable` is `yes`, because it is not true code-wise (we just checked whether `searchable === 'no'`), but I can't find any other way that does not involve using quotation marks 🤔.

Reply:
I also find it weird  why are we doing string matching for `no` when a boolean makes more sense.